blog/Getting Started with ANP.md

```diff
@@ -0,0 +1,270 @@
+
+4. Traffic from the `monitoring-ns` towards any namespace is always allowed, for similar reasons
+   as 3.
+
+5. Traffic from the sales tenant namespaces (or the accounting tenant namespace) towards the
+   `monitoring-ns` is _not_ allowed, for the same reasons as 2.
+
+---
+
+This looks great from the admin's perspective, but what about namespace owners/developers? What
+if, say, the sales team decides that they are okay with sharing data to the accounting team, but
+only through the web application packaged with pre-defined queries instead of the db instances?
+The overridability nature of `BaselineAdminNetworkPolicy` makes this task a breeze.
+Without having to raise tickets to cluster admins for allow rule creation, in the current
+framework the sales team can actually define their own K8s NetworkPolicy for self-serviced
+declaration of the applications that they want to expose to other namespaces:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-access-to-web
+  namespace: s2
+spec:
+  podSelector: {}
+  ingress:
+    - from:
+      - namespaceSelector:
+          matchlabels:
+            kubernetes.io/metadata.name: a1
+```
+
+If the `allow-access-to-web` NetworkPolicy is created in `s2` (the AdminNetworkPolicy and
+BaselineAdminNetworkPolicy from the previous section still exist at the cluster level), the
+security posture of that namespace will change fundamentally compared the the diagram above.
+In particular, ingress traffic from namespace `a1` (labeled 6 in the figure below) will be
+able to connect to `s2` pods since it is explicitly allowed in the NetworkPolicy, which is
+evaluated before any `BaselineAdminNetworkPolicy` rules.
+
+![Diagram3](Diagram3.png)
+
+On the other hand, there's a caveat that many will find surprising: traffic from `s1`, the
+other sales tenant namespace, will be blocked(7) when it reaches the pod in `s2`. Did we not
+create a baseline allow rule for each namespace in the same tenant to connect to each other?
+Yes we did. But as mentioned before, the AdminNetworkPolicy API does not change the behavior
+of NetworkPolicy objects. The `allow-access-to-web` NetworkPolicy, in this case, can be read
+as "all pods in s2 are isolated in the ingress direction, except for traffic from a1 which is
+allowed". So connections initiated from `s1` to `s2` will be dropped due to this implicit
+isolation effect before the `BaselineAdminNetworkPolicy` rules are even evaluated.
+
+The remedy is simple enough though: the `s2` namespace owner (presumably also owns `s1`) can
+simply add an additional ingress rule to the `allow-access-to-web` NetworkPolicy to allow
+traffic from `s1`. The important aspect here though, is that once a NetworkPolicy is applied
+to a pod, it will control it security posture at entirety, unless a `AdminNetworkPolicy` rule
+overrides it from a priority above. `BaselineAdminNetworkPolicy` rules are only effective in
+the absence of those policies.
```

Comment:
Maybe have a quick wrap up paragraph with details on how to get involved in the group?

Reply:
Will work on this